src/__experimental__/components/input-decorator-bridge/input-decorator-bridge.component.js

```diff
@@ -33,24 +31,22 @@ const InputDecoratorBridge = InputDecorator(InputLabel(InputValidation(InputIcon
     render() {
       const { className, ...inputProps } = this.inputProps;
       inputProps.inputRef = this.props.inputRef;
+      inputProps.inputIcon = this.props.inputIcon;
       delete inputProps.ref; // ref is added by decorator, but we would like to move away from needing it
       if (typeof this.props.formattedValue === 'string') inputProps.value = this.props.formattedValue;
 
       return (
         <div className={ this.classes() }>
-          { this.labelHTML }
           <div { ...this.fieldProps }>
             <Textbox { ...inputProps } leftChildren={ this.props.leftChildren }>
               { this.props.children }
             </Textbox>
-            { this.props.inputIcon && this.inputIconHTML(this.props.inputIcon) }
           </div>
           { this.validationHTML }
-          { this.fieldHelpHTML }
         </div>
       );
     }
   }
-))));
+));
 
 export default InputDecoratorBridge;
```

Comment:
Just a general comment, I thought we were trying to move away from the `Decorator` approach?

Reply:
We are - this is a temporary bridge between the new and the old. https://github.com/Sage/carbon/blob/0db0e55c03dff9c8c865dd6ec87ae52ecfcb529a/src/__experimental__/components/input-decorator-bridge/input-decorator-bridge.component.js#L8-L9